package.json

```diff
@@ -8,8 +8,8 @@
     "start-watch": "nodemon index.js",
     "lint": "standard",
     "pretest": "npm run lint",
-    "test": "jest",
-    "test-watch": "jest --watch"
+    "test": "jest --forceExit",
```

Comment:
We are diverging on script names again. I recall some discussion about standardising on script names.
test
test:unit
test:lint
watch:server
watch:unit
I can't find it documented anywhere though, but I think it would make sense for each project to use the same script names for the same actions.

Reply:
I've standardised all the script names (both `scripts/..` and these ones in `package.json`) in another PR: https://github.com/DEFRA/mine-support-calculation-service/pull/4. I'm holding off on requesting a review of that until this PR is merged, at which point I'll rebase and ensure script changes from this branch are incorporated into the new scripts.